js/runner.go

```diff
@@ -195,6 +195,35 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		BPool:          bpool.NewBufferPool(100),
 		Samples:        samplesOut,
 	}
+
+	opts := vu.Runner.Bundle.Options
+	tags := opts.RunTags.CloneTags()
+
+	if opts.SystemTags.Has(stats.TagVU) {
+		tags["vu"] = strconv.FormatInt(vu.ID, 10)
+	}
+	if opts.SystemTags.Has(stats.TagIter) {
+		tags["iter"] = strconv.FormatInt(vu.Iteration, 10)
+	}
```

Comment:
This seems superfluous? It's always going to be 0 here, right?

Reply:
Hmm wondering if we should set any of these tags here at all? After all, `vu` and `iter` don't make sense in `setup()` and `teardown()`, and they set their own `group`... Maybe we should just have `opts.RunTags.CloneTags()` here?

Reply:
fixed in 057f106 
maybe add this as a breaking change ?